### DISCUSSION TITLE
Hide and show a forum

### DISCUSSION THREAD

**Discussion by norschu** [DNNCommunity/Dnn.CommunityForums#154] [Q&A] [answered]
Hello,
Is there a possibility that a forum is only displayed for certain users? So far I can only show or hide the entire forums module for users.

**Comment by johnhenley** [comment, marked as answer]
Hi norschu. Hiding (and more!) is done in the forums control panel rather than in module settings. Security is assigned by DNN security
role to forum groups and can be inherited or overridden for individual forums within a forum group. 

 ![image](https://user-images.githubusercontent.com/9553126/202202921-177efe87-f59a-4019-b09d-6f1e3e5d094e.jpeg)

**Reply by norschu** [reply]
Many thanks for the answer. Now it's working. 🙏
